nautobot/core/templates/generic/object_detail.html

```diff
@@ -105,3 +106,21 @@ <h1>{% block title %}{{ object }}{% endblock %}</h1>
         </div>
     </div>
 {% endblock content %}
+
+
+{% block javascript %}
+<script src="{% static 'clipboard.js/clipboard-2.0.6.min.js' %}"
+    onerror="window.location='{% url 'media_failure' %}?filename=clipboard.js/clipboard-2.0.6.min.js'">
+</script>
+<script>
+    var clipboard = new ClipboardJS('.btn');
+
+    clipboard.on('success', function (e) {
+        console.log(e);
+    });
+
+    clipboard.on('error', function (e) {
+        console.log(e);
+    });
+</script>
```

Comment:
Is any of this debugging-only code that should be removed before merge?

Reply:
This is copied from somewhere else in Nautobot, I can remove the console.log